unit-tests/live/live-common.h

```diff
@@ -31,7 +72,7 @@ inline rs2::device_list find_devices_by_product_line_or_exit( int product )
                   << std::endl;
         exit( 1 );
     }
-
+    test::log.d( "found", devices_list.size(), "devices:", repr( devices_list ));
```

Comment:
The enum is not a real char it actually prints the asci code of the char and not the char itself.

See [here](https://rsjenkins.iil.intel.com/job/LRS_libci_pipeline/1792/artifact/win10/win64/static/RelWithDebInfo/unit-tests/test-live-syncer-throughput.log)

-68- found <device: Intel RealSense D455 (S/N: 013322250065  FW: 05.12.15.50  UNLOCKED  on USB3.2)>